modules/table/src/test/java/org/apache/ignite/internal/table/distributed/raft/snapshot/PartitionAccessImplTest.java

```diff
@@ -204,4 +208,12 @@ void testAddWriteCommitted() {
 
         verify(indexStorage, never()).put(eq(binaryRow), eq(rowId));
     }
+
+    private static MvPartitionStorage createMvPartition(MvTableStorage tableStorage, int partitionId) {
+        CompletableFuture<MvPartitionStorage> createMvPartitionFuture = tableStorage.createMvPartition(partitionId);
+
+        assertThat(createMvPartitionFuture, willCompleteSuccessfully());
+
+        return createMvPartitionFuture.join();
```

Comment:
Probably should be `get` with timeout

Reply:
It is not necessary because above we make sure that the future completed.

`assertThat(createMvPartitionFuture, willCompleteSuccessfully());`

Reply:
how about getNow() then?

Reply:
I do not think that this is necessary, the future is already definitely completed.